src/Peering/Peering.Test/Peering.Test.csproj

```diff
@@ -11,7 +11,8 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.Azure.Management.Peering" Version="0.9.0-preview" />
+    <PackageReference Include="Microsoft.Azure.Management.Peering" Version="0.10.0-preview" />
+    <PackageReference Include="Microsoft.Rest.ClientRuntime" Version="2.3.20" />
```

Comment:
@rileymckenna you can remove this dependency -- we have it defined already in a common `.targets` file that all `.csproj` files reference